lib/active_fulfillment/models/dotcom_distribution/post_item.rb

```diff
@@ -118,7 +118,9 @@ def item_to_xml(xml)
           xml.send(:"description") do
             xml.cdata self.description
           end
-          xml.send(:"quantity", self.quantity, inject_nil(self.quantity))
+          if quantity
```

Comment:
I don't see `quantity` in the docs, though I do see `package-qty`. Also, if it's absent shouldn't we use `inject_nil(self.quantity)` (or rather `package_qty`)?